ArthurL.txt

```diff
@@ -3,7 +3,7 @@
 Hi, my name is Arthur. 
 
 
-Male, human, and mortal are three words that describe me. 
+Radagast the Brown, Radagast the Simple, and Radagast the Fool would be three words that describe me perfectly. 
 
 
-My favorite hobby is learning. This manifests itself in different ways. I have a passion for traveling, cooking, mathematics, chess, logic, sports, world music, different cultures, different ways of thinking, spending time with anyone and anything with more intelligence than me. I believe that learning, like everything else in life, needs to be shared in order to be fulfilling.
+My favorite hobby is conversing with birds. My greatest concern in this world is with the olvar and kelvar. I am wiser than any man in all things concerning herbs and beasts. I dwell in Rhosgobel by the Vale of Anduin. I am never a traveller unless driven by great need. I rarely wash my hair.
```

Comment:
You're going to love Luis, who teaches back-end. It's basically five weeks of LOTR references.